datafusion/sqllogictest/test_files/csv_files.slt

```diff
@@ -63,3 +63,76 @@ id6 value"6
 id7 value"7
 id8 value"8
 id9 value"9
+
+
+# Read partitioned csv
```

Comment:
`partitioned_csv.rs` did not have any unit tests so I created a new one to test reading from partitioned csv files.